draft-ietf-teep-architecture.md

```diff
@@ -92,8 +92,8 @@ The Trusted Execution Environment (TEE) concept is designed to execute
 applications in a protected environment that enforces that any code 
 within that environment cannot be tampered with, 
 and that any data used by such code 
-cannot be read or tampered with by any code outside that environment,
-including by a commodity operating system (if present).
+cannot be read or tampered with by any code outside that environment in the 
+Rich Execution Environment (REE), including by a commodity operating system (if present).
```

Comment:
This change is not correct, in my opinion.  If you have multiple TEEs, it also means that one TEE cannot be read or tampered with by code in the other TEE.   Please revert.

Reply:
If we want to point out that the one TEE cannot tamper with the other TEE then we should explicitly say that.  I also think that somewhere in the introduction section the concepts REE and TEE need to be mentioned.